applications/luci-app-softether/Makefile

```diff
@@ -0,0 +1,15 @@
+#
```

Comment:
I can not see any acl file under `root/usr/share/rpcd/acl.d/luci-app-softether.json`
I have not tested it now in master. But if this file is not there then the menu entry should not be displaye.

Reply:
I checked on snapshot r15252. It is the other way around. The menu item was shown even on logon screen.
Added a generic ACL reference like in other status page.
Done.